src/main/java/core/basesyntax/supplier/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.enums.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    public static String getRandomColor() {
+        Random random = new Random();
```

Comment:
lets make Random as class level variable, not to create new instance of Random class every time we call getRandomColor() method